constellations/gemini-user/src/achievements/achievements.sass

```diff
@@ -1,2 +1,61 @@
 #view-achieve h1
-  color: red
+	color: red
+
+
+.achieve-card
+	margin-bottom: 96px
+	
+	& h2
+		line-height: 30px
+		font-size: 18px
+		color: #056571
+		border-bottom: 1px solid #222
```

Comment:
It's important to properly order CSS, so take a look at this file: https://github.com/ahsu1230/mathnavigatorSite/blob/master/resources/frontend/css_ordering.md
So border-bottom would come first, then color, then font-size and line-height.